openeo-geotrellis/src/test/scala/org/openeo/geotrellis/MergeCubesSpec.scala

```diff
@@ -121,6 +144,63 @@ class MergeCubesSpec {
     assertTrue(mse < 0.1)
   }
 
+
+  @ParameterizedTest
+  @MethodSource(Array("testMergeCubesTiledNodataArguments"))
+  def testMergeCubesTiledNodata(aggregateR: AggregationType.Value,
```

Comment:
Consider adding some comments about how the test works.

Reply:
done